node_modules/oae-config/tests/test-config.js

```diff
@@ -1162,5 +1163,34 @@ describe('Configuration', function() {
             });
         });
 
+        /**
+         * Test that verifies that the guests tenant config can be updated just like any other tenant
+         */
+        it('verify the guests tenant config can be updated', function(callback) {
+            // Update the discussion visibility on the guests tenant
+            var config = {
+                'oae-discussions/visibility/discussion': 'private',
+                'oae-principals/recaptcha/enabled': false
+            };
+            ConfigTestUtil.updateConfigAndWait(globalAdminRestContext, 'guests', config, function(err) {
```

Comment:
+1 for making that a constant. That would also take away the potential issue of the alias being changed along the line.